src/stylesheets/components/_header.scss

```
@@ -125,7 +125,7 @@ $z-index-overlay: 400;
   @include u-pin("all");
   position: fixed;
   background: color("black");
-  opacity: opacity(0);
+  opacity: opacity(70);
```

```suggestion
  opacity: opacity(0);
```

Let's revert this change because our background color is semi-transparent.

Which means we'll have to update `line:134` below.

```scss
&.is-visible {
  opacity: opacity(100);
  visibility: visible;
}
```

---

updated